themes/default/layouts/case-studies/list.html

```diff
@@ -31,6 +31,25 @@ <h5 class="leading-none">
                     </article>
                 </li>
             {{ end }}
+
+            <li class="m-0 p-0">
+                <article class="md:flex items-start">
+                    <div class="mb-8 md:mb-0 md:pt-2 px-8 md:w-1/3">
+                        <img class="md:block w-40 md:w-48 mx-auto" src="/logos/customers/kenshoo-wordmark.svg" alt="Kenshoo logo">
+                    </div>
+                    <div class="md:w-2/3">
+                        <h5 class="leading-none">
+                            <a class="link" href="{{ relref . "/blog/kenshoo-migrates-to-aws-with-pulumi" }}">
+                                Kenshoo Migrates to AWS with Pulumi
+                            </a>
+                        </h5>
+                        <p class="mb-0">
+                            Kenshoo selected Pulumi as our IaC framework of choice due to its native programmatic nature,
```

Comment:
These are written from our POV, so 

```suggestion
                            Kenshoo selected Pulumi as their IaC framework of choice due to its native programmatic nature,
```